.travis.sh

```diff
@@ -9,7 +9,13 @@ php --version
   cd $(mktemp -d)
   curl https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
 )
-hhvm /usr/local/bin/composer install
+
+runtime=$(hhvm --php -r "echo HHVM_VERSION[0] === '3' ? 'hhvm' : 'php';")
```

Comment:
Should this be:
```suggestion
runtime=$(hhvm --php -r "echo HHVM_VERSION[0] === '3' ? 'php' : 'hhvm';")
```
?

Reply:
No, this is the right way around.
If your hhvm version starts with a 3,
hhvm should be in charge of running composer.
If you are hhvm 4 or up,
let php do it.
Hhvm 4 can't run composer.

Reply:
Gotcha, thanks.